### PR TITLE
ADAM6217 device class

### LINKED ISSUE #2: Prepare a Device Class for ADAM-6217-AE
The model ADAM-6217-AE contains 8 Analogue Inputs (here's [a website with a datasheet](http://eu.mouser.com/ProductDetail/Advantech/ADAM-6217-AE/)).

### PR DESCRIPTION
**Pull request by Patfra**
Closes #2 #9 

- added class for ADAM-6217 Module
- added multiple Device Classes runinning on single Device Server mechanism

### CODE REVIEW

**Review by LJBD — CHANGES_REQUESTED** [review]
Please rebase this branch onto the master branch and resolve conflicts.

**Comment by Patfra** [comment]
I have resolved conflicts.

File: adam/adam_6217.py [hunk 1]
```diff
@@ -0,0 +1,635 @@
...
+            self.set_state(DevState.FAULT)
+            self.set_status("Exception caught while connecting to device:"
+                            + "\n%s" % e)
+        self.set_state(DevState.ON)
+        self.set_status("Connected do device with IP: "
+                        + str(self.DeviceAddress))
+        self.info_stream("Connected do device with IP: " + str(
+             self.DeviceAddress))
+
+    @command(dtype_in = int, doc_in='Channel number')
```
Comment (LJBD) [inline adam/adam_6217.py]: The equal sign shouldn't be surrounded with spaces when you're setting a default argument to a method.

File: adam/adam_6217.py [hunk 2]
```diff
@@ -0,0 +1,635 @@
...
+        dtype='str',
+        default_value="192.168.120.56",
+        doc = "This property contains an IP address of device"
+    )
+
+    # ----------
+    # Attributes
+    # ----------
+
+    AnalogInput_0 = attribute(
```
Comment (LJBD) [inline adam/adam_6217.py:108]: Please add descriptions to the attributes.

**Review by LJBD — CHANGES_REQUESTED** [review]

**Review by LJBD — APPROVED** [review]

**Pull request merged**